roles/network-multus/templates/ovs-cni.yml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
I don't like that we have the definition duplicated here, the same for ovs-cni. Would it be possible to copy it from https://github.com/kubevirt/ovs-cni/tree/master/examples on the fly?

Reply:
Unless there's a single source that is guaranteed to match OCP requirements (for example, /master/examples/... may now, but not in the future), I feel like we best maintain our own templates locally

Reply:
all right